src/amo/components/MastHead.scss

```diff
@@ -0,0 +1,66 @@
+@import "~amo/css/inc/vars";
+
+#tabzilla {
+  left: 14px;
+  position: absolute;
+}
+
+.MastHead {
+  background: $header-background;
+  min-height: 100px;
+  position: relative;
+}
+
+.MastHead-header {
+  padding-top: 80px;
+  padding-bottom: 30px;
+  text-align: center;
+}
+
+.MastHead-title {
+  align-items: center;
+  color: $header-font-color;
+  display: flex;
+  font-size: 24px;
+  justify-content: center;
+
+  &:link {
+    font-size: 24px;
+    text-decoration: none;
+  }
+
+  &::before {
+    background: url(../img/icons/fox-puzzle.svg) no-repeat 50% 50%;
+    content: '';
+    display: inline-block;
+    height: 50px;
+    margin-right: 10px;
+    width: 50px;
+  }
+}
+
+.AccountButton.button {
+  position: absolute;
```

Comment:
Any chance we can come up with some rules around CSS property order? I personally am a huge fan of alphabetising them so it's easy to scan/find things.

If that's too obsessive for everyone else it's okay, but I find it really makes CSS properties, which otherwise are quite inconsistent across different developers, very consistent and sense-making.